Library/Formula/vile.rb

```diff
@@ -0,0 +1,23 @@
+class Vile < Formula
+  desc "Vi Like Emacs"
+  homepage "http://invisible-island.net/vile/vile.html"
+  url "http://invisible-island.net/datafiles/release/vile.tar.gz"
+  version "9.8"
+  sha256 "dea31c023aa7a90271056c712c826a5d16d19ad61df5cf5f299aa1025af280d6"
+
+  depends_on "reflex"
```

Comment:
Sorry I'm so helpless with git - now I'm getting an error about fast-forwards:

    /usr/local$ git push -u timbck2 adding_vile
    To https://github.com/timbck2/homebrew.git
     ! [rejected]        adding_vile -> adding_vile (non-fast-forward)
    error: failed to push some refs to 'https://github.com/timbck2/homebrew.git'
    hint: Updates were rejected because the tip of your current branch is behind
    hint: its remote counterpart. Integrate the remote changes (e.g.
    hint: 'git pull ...') before pushing again.
    hint: See the 'Note about fast-forwards' in 'git push --help' for details.